unified-codes/tools/scripts/dgraph/data/v1/products.json

```diff
@@ -94,7 +94,7 @@
           "type": "code_who_atc"
         },
         {
-          "value": 51342501,
+          "value": "51342501",
```

Comment:
This isn't a `code_rxnav` 🙀 ! I feel lied to...But the change makes sense.

Reply:
![run](https://user-images.githubusercontent.com/43223637/110705689-9f3ad580-825b-11eb-8e82-bcac9d2da9d6.gif)